python-connectors/open-weather-map_once-call-data/connector.py

```diff
@@ -0,0 +1,54 @@
+from dataiku.connector import Connector
+from openweathermap_utils import OpenWeatherMapAPI, CacheHandler
+from openweathermap_utils.utils import get_cache_location_from_configs
+
+
+class OpenWeatherMapConnector(Connector):
+
+    def __init__(self, config, plugin_config):
+        Connector.__init__(self, config, plugin_config)
+
+        preset_config = self.config.get("preset_config")
+
+        self.cache_location = get_cache_location_from_configs(
+            cache_location=self.plugin_config.get('cache_location'),
```

Comment:
Be consistent with the quotes (use double quotes for all strings)

Reply:
Definitely right, done in next PR